components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java

```diff
@@ -2803,6 +2851,40 @@ private LocalAndOutboundAuthenticationConfig getLocalAndOutboundAuthenticationCo
         }
     }
 
+    private AppRoleMappingConfig[] getAppRoleMappingConfigurations(int applicationId, Connection connection)
+            throws SQLException {
+
+        PreparedStatement getAppRoleMappingConfigPrepStmt = null;
+        ResultSet getAppRoleMappingConfigResultSet = null;
+        List<AppRoleMappingConfig> appRoleMappingConfigList = new ArrayList<>();
+
+        try {
```

Comment:
Can use try with resources here as well.

Reply:
Addressed in 3da9288685f309f6395405363622af44c39f34c2